time-tracker/src/components/task-adder.jsx

```diff
@@ -0,0 +1,22 @@
+import { useDispatch } from "react-redux";
+import { addNewTask } from "../redux/reducers.jsx";
+import { useState } from "react";
+
+export default function TaskAdder() {
```

Comment:
Adder is not a known term, as you'd see naming things is sometimes one of the most difficult things.

How about NewTaskInput?